imagestreams/postgresql-rhel.json

```diff
@@ -10,217 +10,203 @@
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
           "version": "13"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhscl/postgresql-13-rhel7:latest"
+          "name": "registry.redhat.io/rhel8/postgresql-13:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "12-el8",
+        "name": "15-el8",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL 12 (RHEL 8)",
+          "openshift.io/display-name": "PostgreSQL 15 (RHEL 8)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 12 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 15 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
-          "version": "12"
+          "version": "15"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhel8/postgresql-12:latest"
+          "name": "registry.redhat.io/rhel8/postgresql-15:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "12-el7",
+        "name": "13-el9",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL 12 (RHEL 7)",
+          "openshift.io/display-name": "PostgreSQL 13 (RHEL 9)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 12 database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 13 database on RHEL 9. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
-          "version": "12"
+          "version": "13"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhscl/postgresql-12-rhel7:latest"
+          "name": "registry.redhat.io/rhel9/postgresql-13:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "12",
+        "name": "15-el9",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL (Ephemeral) 12",
```

Comment:
I guess, the name should be the same `PostgreSQL (Ephemeral) 12` and not `PostgreSQL 12 (Ephemeral)`.

Reply:
This seems to me like the original version being incorrect, as all other display names have same format, except for the Ephemeral one.

Reply:
Ephemeral streams are now removed.